### DISCUSSION TITLE
Ability to apply saved Custom Pin Mappings to other similar boards

### DISCUSSION THREAD

**Discussion by l3giona1re** [TeamOpenFIRE/OpenFIRE-App#14] [Ideas]
**Request**: Save custom pin mapping and have option to upload this to a new board that has the same physical pin layout.

**Background:** Right now the app lets me save the custom pin mapping for a board, however, if I had to reset that board for what ever reason, when I load the firmware back on, I have to do the custom pin mapping all over again.
Also when I'm making multiple guns and want the same custom mapping, I would have to screencapture the pin mapping, and have that screencapture image next to OF app to replicate.

**Comment by SeongGino** [comment]
Good idea, see #15
